Assets/_EscapeRoom/Scripts/UICrosshair.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace EscapeRoom
+{
+    public class UICrosshair : MonoBehaviour
+    {
+        public enum CrosshairStates
+        {
+            None,
+            Interactable,
+        }
+
+        [SerializeField] private Image _crosshairImage = null;
+        [SerializeField] private Sprite _noneCrosshairSprite = null;
```

Comment:
is this when theres no crosshair? like its disabled?

Reply:
ah okay, nvm i understand. I think for naming
i would call it nonInteractableCrosshairSprite

Reply:
I added a default crosshair which is just a dot. I renamed this to _default to make it more clear. You can set the image to no sprite which will remove the default crosshair